codice/ddf
# Fix SolrCloud Docker volume mounting and removing

#### What does this PR do?
Fixes an incorrect volume names on delete and adds a Zookeeper datalog volume due to a docker image change.

#### Who is reviewing it? 


#### Select relevant component teams: 

@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.

@rzwiefel

#### How should this be tested?

Start SolrCloud, ingest data and verify it is searchable, bring it down and back up, and verify data is still searchable.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: n/a

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
